Sources/OutputTarget.swift

```diff
@@ -32,7 +32,7 @@ import Darwin.C
 
 private func getEnvValue(key: String) -> String? {
     let value = getenv(key)
-    return value != nil ? String.fromCString(value) : nil
+    return value != nil ? String(cString: value!) : nil
```

Comment:
So it's saying if its not nil ``String(cString: value!) else nil`` then?

Would you like me to add that change?